lua/nvim-treesitter/configs.lua

```diff
@@ -28,6 +28,21 @@ local config = {
       is_supported = function(lang)
         return queries.get_query(lang, 'locals')
       end
+    },
+    textobj = {
+      enable = true,
+      disable = {},
+      is_supported = function(_) return true end,
```

Comment:
maybe this should check the the query for textObj exists (either the locals.scm or the textObj.scm) ?